src/components/tools/geometry-tool/geometry-content.tsx

```diff
@@ -1087,6 +1088,19 @@ export class GeometryContentComponent extends BaseComponent<IProps, IState> {
           if (hasSelectionModifier(evt)) {
             geometryContent.deselectElement(id);
           }
+
+          // If a line is selected and a control point is pressed, deselect the line so the line slope can be changed
+          if (isMovableLineControlPoint(point)) {
```

Comment:
I would suggest replacing the body of the `if` with a function call to something like `handleControlPointClick()` and then move the body of the function to `jxg-movable-line.ts`, as it's essentially an implementation detail.

Reply:
Good suggestion, helps keep down the size of this file, too. Accepted your other suggestion through the Github UI which was cool!